tests/integration/internal/receivermock/receiver_mock.go

```diff
@@ -122,6 +122,41 @@ func (client *ReceiverMockClient) GetMetricsSamples(
 	return metricsSamples, nil
 }
 
+type LogsCountResponse struct {
+	Count uint
+}
+
+func (client *ReceiverMockClient) GetLogsCount(t *testing.T, metadataFilters MetadataFilters) (uint, error) {
+	path, err := url.Parse("logs/count")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
It might be worth thinking about putting this e.g. in the `ReceiverMockClient` struct itself (or as package wide unexported variable parsed at `init`)

Reply:
I'll leave a TODO for it.